app/middleware/server_id.rb

```diff
@@ -22,7 +23,27 @@ def call(env)
     response_headers[HEADER_KEY] = SERVER_ID
     response_headers.merge! CHEEKY_HEADERS
     response_headers.merge! SECURITY_HEADERS
+    response_headers.merge! throttle_headers(env)
 
     response
   end
+
+  private
+
+  def throttle_headers(env)
+    throttler_name = 'req/ip'
```

Comment:
Perhaps, using a constant would be better here. Maybe even a shared constant between the definition of `throttle` and this.